pkg/scheduler/algorithmprovider/registry.go

```diff
@@ -50,17 +50,19 @@ const ClusterAutoscalerProvider = "ClusterAutoscalerProvider"
 // Registry is a collection of all available algorithm providers.
 type Registry map[string]*schedulerapi.Plugins
 
+// ApplyFeatureGates applies feature gates to registry
+func (r Registry) ApplyFeatureGates() Registry {
+	for _, config := range r {
```

Comment:
for loop would range the map randomly, would this affect the result of the feature gates?

Reply:
Thx,  agree, change done.